src/shortcuts/__tests__/newThoughtOrOutdent.js

```diff
@@ -1,10 +1,12 @@
-import { store } from '../../store'
-import { keyDown } from '../../shortcuts'
+import { keyboardHandlers } from '../../shortcuts'
 import { importText } from '../../action-creators'
 import { RANKED_ROOT, ROOT_TOKEN } from '../../constants'
 import { exportContext } from '../../selectors'
 import { noop } from 'lodash'
 import { initialize } from '../../initialize'
+import { createTestStore } from '../../test-helpers/createTestStore'
+
+const store = createTestStore()
```

Comment:
This should probably be assigned in `beforeEach`, otherwise data could carry over between tests.